projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -36,7 +39,7 @@ class Jetpack_Memberships {
 	 *
 	 * @var string
 	 */
-	private static $button_block_name = 'recurring-payments';
+	private static $button_block_name = 'recurring-payments-button';
```

Comment:
This is used for a css class name when rendering a deprecated v1 button, changing this could break any 3rd party css styles. Perhaps it ought to remain with it's old value, or `deprecated_render_button_v1` changed.

It's also used to load 3rd party assets in `render_button` but that's going to have to use a different value anyhow